### PR TITLE
Github Enterprise support

### PR DESCRIPTION
**Pull request by chernjie**
image

### CODE REVIEW

**Comment by chernjie** [comment]
Previously image

**Comment by chernjie** [comment]
@philschatz I have added a textbox to add enter a URL

**Comment by philschatz** [comment]
Thank you! Unfortunately I do not use GitHub Enterprise so it has not been a priority but thank you for getting it working.

File: src/components/login-modal.jsx [hunk 1]
```diff
@@ -65,6 +82,16 @@ export default React.createClass({
...
+            <h4>"GitHub Enterprise Endpoint"</h4>
+            <p>
+              If you need to set a custom API endpoint:<br/>
+              <BS.Input
+                type='text'
+                defaultValue={defaultRootURL}
+                placeholder='Enter GitHub API URL, e.g. https://github.example.com/api/v3'
+                ref='rootURL'
+              />
+            </p>
```
Comment (chernjie) [inline src/components/login-modal.jsx]: @philschatz the new changes from 94c043d uses `BS.FormControl`, perhaps it's better to standardise them?

Comment (chernjie) [inline src/components/login-modal.jsx]: Having said that, due to 94c043d I kept getting `token/rootURL has no method getValue defined`.

**Pull request merged**